updatehub/src/states/actor/remote_install.rs

```diff
@@ -23,6 +23,7 @@ impl Handler<Request> for super::Machine {
             return match res {
                 Response::InvalidState(_) => MessageResult(res),
                 Response::RequestAccepted(_) => {
+                    crate::logger::buffer().lock().unwrap().start_logging();
```

Comment:
This is what I'd like to avoid to handle manually and try to hide inside the logger module API.

Reply:
This `Mutex` locking you're tagging here is not the same `Mutex` you mentioned before and this one can be more easily avoided. However when we respond the [http_api log](https://github.com/UpdateHub/updatehub/blob/next/updatehub/src/http_api.rs#L64) request we need to access the `Arc<Mutex<T>>` and not the content itself, because `Serialize` is implemented for `Arc<Mutex<T>>` but not for `MutexGuard<'_, T>`.

We could offer two methods for accessing the buffer, one that would block and return the `MutexGuard` and one that would give the hole `Arc<Mutex<T>>` to be used for the log request. What do you think about it?